presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -98,6 +98,7 @@ General Aggregate Functions
 .. function:: max(x) -> [same as input]
 
     Returns the maximum value of all input values.
+    NaN values would be ignored in the input.
```

Comment:
This is not quite accurate. When all input values are NaN, the result is NaN, no?

Reply:
My impression is that the behavior is the following:

```
Returns the maximum value of all input values. 

When x is a REAL or DOUBLE returns NaN if all input values are NaN. Ignores NaN values otherwise.
```

It might be helpful to include some examples.

Reply:
Thanks for bearing with me on this PR. Done amending